src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -459,3 +460,22 @@ def find_child_collection(parent, *args, **kwargs):
     if collection is None:
         raise CLIError("collection '{}' not found".format(collection_path))
     return collection
+
+
+def check_connectivity(url='https://example.org', max_retries=0, timeout=1):
```

Comment:
Is https://example.org reliable? Should we use azure.microsoft.com or other more reliable url to test the connectivity?